docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -88,27 +129,48 @@ wp-env --version
 > 1.6.0
 ```
 
-The `wp-env` script is used to create a Docker WordPress environment, to use start from your plugin directory—if you are following the create block tutorial, this would be in the generated directory.
+The `wp-env` script is used to create a Docker WordPress environment. You can use this script to start an environment with your plugin activated by running it from the directory containing your plugin. For example if you are following the create block tutorial, this would be in the generated directory like so:
 
 ```sh
+npx @wordpress/create-block starter-block
+cd starter-block
 wp-env start
 ```
 
 You can access your environment in your browser at: [http://localhost:8888/](http://localhost:8888/), the default username is `admin` and default password is `password`. For more information controlling the Docker environment see the [@wordpress/env package readme](/packages/env/README.md).
 
```

Comment:
Let's add some Docker info here.
Another option would be to open the browser through Docker. Go to the Docker Dashboard, then click the stack and select a container to open.
---- or something like that. Then the following information can be added after it.... "For more information about controlling the Docker environment see the...."


Full text as I see it:
You can access your environment in your browser at: [http://localhost:8888/](http://localhost:8888/), the default username is `admin` and default password is `password`. Another option would be to open the browser through Docker. Go to the Docker Dashboard, then click the stack and select a container to open.
For more information about controlling the Docker environment see the [@wordpress/env package readme](/packages/env/README.md).

Reply:
I don't think we need to add Docker info. It is confusing and gives them no additional benefit than just typing that address in their browser. Also, the directions only work if you're running Docker Desktop, which some like myself do not have that interface, I just run the daemon service.

Reply:
Ahhh gotcha! I am letting it go..:)